ocrd_tesserocr/crop.py

```diff
@@ -87,8 +88,12 @@ def process(self):
                                 min_x, max_x, min_y, max_y)
                 
                 page_image = self.workspace.resolve_image_as_pil(page.imageFilename)
-                dpi = page_image.info.get('dpi', (300,300))[0]
-                zoom = 300 / dpi
+                dpi = page_image.info.get('dpi', (0,0))[0]
```

Comment:
Thanks! I will add that to the PR as soon as my HDD is operable again.

> BTW: Should we offer some unified resolution derived from x- and y-Resolution?

With pleasure!

And we need to make the fallback (be it `1` or another) be decisive and stable, so we have a means to known when _not_ to override Tesseract's own DPI guessing.

Reply:
So @kba I guess I will wait for you to release a new version of `OcrdExif`, or maybe even to ensure annotation in `Page` attributes, as outlined below? Until then this can stay the way it is, right?

Reply:
> release a new version of OcrdExif

You mean with the unified resolution?

>  Until then this can stay the way it is, right?

I would really use OcrdExif even now because your code won't reliably handle JPEG or PNG resolution. But it's not essential for the PR here, can be changed later if it's not flexible enough.

Reply:
> You mean with the unified resolution?

Yes. And especially the stable (guaranteed) default.

> I would really use OcrdExif even now because your code won't reliably handle JPEG or PNG resolution. But it's not essential for the PR here, can be changed later if it's not flexible enough.

Okay, than I will change it now and we will already be able to gather experience with `OcrdExif`. I will only use `xResolution` and rely on `1` as being unset for now.

Reply:
Oh, and one more thing, @kba: It looks to me like the code which determines `resolutionUnit` for JPEG does not convert the internal JFIF integers to strings like for the other formats. See [here](https://github.com/python-pillow/Pillow/blob/1c57a41ececc4529520a423b9bec13c78927cb25/src/PIL/JpegImagePlugin.py#L86) for how Pillow creates `jfif_unit`.

Reply:
Confirmed! I will open an issue about it.

Reply:
done: OCR-D/core#256